tests/udp/test_raw_traffic_with_fixed_udp_ports.py

```diff
@@ -0,0 +1,78 @@
+from abstract_open_traffic_generator.flow import *
+import pytest
+
+from abstract_open_traffic_generator import flow
+import utils
+
+
+@pytest.mark.parametrize('packets', [1000])
```

Comment:
Let's get rid of packets and size fixtures as they increase test runtime a lot. I did the same for ethernet/* scripts.
We'll have separate, dedicated tests to cover these.

Reply:
Have removed multiple iterations but has kept a single value instead of hardcoding inside the test